net.eudic.dict.yaml

```diff
@@ -0,0 +1,74 @@
+app-id: net.eudic.dict
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+separate-locales: false
+rename-icon: eudic
+rename-desktop-file: eudic.desktop
+command: eudic 
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --socket=pulseaudio
+  - --device=all
+  - --env=XCURSOR_PATH=/run/host/user-share/icons:/run/host/share/icons:/usr/share/icons
+
+modules:
+  - name: krb5
+    buildsystem: autotools
+    config-opts:
+      - --disable-rpath
+      - --enable-shared
+    subdir: src
+    sources:
+      - type: archive
+        url: 'https://kerberos.org/dist/krb5/1.21/krb5-1.21.3.tar.gz'
+        sha256: b7a4cd5ead67fb08b980b21abd150ff7217e85ea320c9ed0c6dadd304840ad35
+        x-checker-data:
+          type: anitya
+          project-id: 13287
+          stable-only: true
+          url-template: >-
+            https://kerberos.org/dist/krb5/${version0}.${version1}/krb5-${version0}.${version1}.${version2}.tar.gz
+    
+  - name: eudic
+    buildsystem: simple
+    build-commands:
+      - install -Dm644 ${FLATPAK_ID}.metainfo.xml -t /app/share/metainfo
+      - install -Dm644 eudic.png -t /app/share/icons/hicolor/256x256/apps
+      - install -Dm644 eudic.desktop -t /app/share/applications
+      - install -D eudic.sh /app/bin/eudic
+      - install -D apply_extra -t /app/bin
+    sources:
+      - type: script
+        dest-filename: apply_extra
+        commands:
+          - bsdtar --to-stdout -xf eudic.deb data.* | bsdtar -xf -
+          - mv usr/share/eusoft-eudic ./
+          - rm -r *.deb usr
+          - cd eusoft-eudic 
+          - rm libg* lib/libg* libmount* libkrb* libk5* libwayland* libatk* libudev.so.1 libmpg123.so.0 
+
+      - type: script
+        dest-filename: eudic.sh
+        commands:
+          - export LD_LIBRARY_PATH="/app/extra/eusoft-eudic/:$LD_LIBRARY_PATH"
+          - exec /app/extra/eusoft-eudic/eudic "$@"
+
+      - type: file
+        path: net.eudic.dict.metainfo.xml
+
+      - type: file
+        path: eudic.png
+
+      - type: file
+        path: eudic.desktop
+
+      - type: extra-data
+        filename: eudic.deb
+        only-arches: [x86_64]
+        url: https://static.frdic.com/pkg/eudic.deb?v=2025-01-26
```

Comment:
is this a stable URL or will it change?

Reply:
Seems this currently is no effect, but it's got from the official website.
Let's keep it.

Reply:
So needs to update the sha256.  
I think it's no a problem. Some software may use a different link addr for a new version, but they can drop down the old link too.   
Better to add a http checker.

Reply:
Added.

Reply:
you can ask for an exception to automerge if needed, later https://docs.flathub.org/docs/for-app-authors/external-data-checker#automatically-merge-prs